src/Command/CsFixerInitCommand.php

```diff
@@ -36,5 +36,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $destination . '/.' . $template
             );
         }
+
+        return self::SUCCESS;
```

Comment:
I think you have a problem with your phpstan container, because this const is not available before Symfony 5, it failed on Symfony 3 & 4. The composer.lock is targeting the v3.4.42

Reply:
composer.lock does not propagate, if you are only compatible with 3-4 but not 5 you should specify so in the composer.json

Reply:
I have updated it to use an int so that it should be compatible with Symfony 3-5

Reply:
I don't see any reason to keep the composer.lock file in this repository :thinking: